### PR TITLE
Jk/cumulus 2985

### PR DESCRIPTION
**Pull request by Jkovarik**
Changes: 

Updates template to use Core v16.0.0

Removes all references to data-migration1

### CODE REVIEW

File: cumulus-tf/main.tf [hunk 1]
```diff
@@ -47,7 +47,7 @@ data "terraform_remote_state" "data_persistence" {
 
 
 module "cumulus" {
-  source = "https://github.com/nasa/cumulus/releases/download/v15.0.2/terraform-aws-cumulus.zip//tf-modules/cumulus"
+  source = "https://github.com/nasa/cumulus/releases/download/v16.0.0terraform-aws-cumulus.zip//tf-modules/cumulus"
```
Comment (npauzenga) [inline cumulus-tf/main.tf:50]: Missing a "/" I think.

**Review by npauzenga — CHANGES_REQUESTED** [review]
Looks good @Jkovarik. Just one change.

File: cumulus-tf/main.tf (hunk 1 above)
Comment (Jkovarik) [inline cumulus-tf/main.tf:50]:
```suggestion
  source = "https://github.com/nasa/cumulus/releases/download/v16.0.0/terraform-aws-cumulus.zip//tf-modules/cumulus"
```

**Comment by Jkovarik** [comment]
@npauzenga  Fixed.  Thanks for catching that 😓

**Review by npauzenga — APPROVED** [review]
Nice. Thanks @Jkovarik!

**Pull request merged**